### PR TITLE
improve import path

### PR DESCRIPTION
**Pull request by kahirokunn**
Closes https://github.com/rtk-incubator/rtk-query-codegen/issues/27

### CODE REVIEW

File: src/bin/cli.ts [hunk 1]
```diff
@@ -20,7 +21,8 @@ program
   .option('--responseSuffix <name>', 'pass response suffix')
   .option('--baseUrl <url>', 'pass baseUrl')
   .option('-h, --hooks', 'generate React Hooks')
-  .option('--file <filename>', 'output file name (ex: generated.api.ts)')
+  .option('-c, --config <path>', 'pass tsconfig path for resolve path alias')
+  .requiredOption('-f, --file <filename>', 'output file name (ex: generated.api.ts)')
```
Comment (msutkowski) [inline src/bin/cli.ts:25]: I'm not sure if we want to make this required.

File: test/cli.test.ts [hunk 2]
```diff
@@ -5,19 +5,32 @@ import del from 'del';
...
+function cli(
+  args: string[],
+  cwd: string
+): Promise<{ error: ExecException | null; stdout: string; stderr: string; output: string }> {
   return new Promise((resolve) => {
+    const tmpFileName = path.resolve(tmpDir, `${++id}.test.generated.ts`);
     exec(
-      `ts-node -T -P ${path.resolve('./tsconfig.json')} ${path.resolve('./src/bin/cli.ts')} ${args.join(' ')}`,
+      `ts-node -T -P ${path.resolve('./tsconfig.json')} ${path.resolve('./src/bin/cli.ts')} ${[
+        `-f ${tmpFileName}`,
```
Comment (msutkowski) [inline test/cli.test.ts:19]: Same thing, do we want to require a file?

File: package.json [hunk 3]
```diff
@@ -14,14 +14,15 @@
     "prepare": "npm run build && chmod +x ./lib/bin/cli.js",
     "format": "prettier --write \"src/**/*.ts\"",
     "test:update": "lib/bin/cli.js test/fixtures/petstore.json --file test/fixtures/generated.ts -h",
-    "test": "jest"
+    "test": "jest --updateSnapshot --silent=false --verbose"
```
Comment (msutkowski) [inline package.json:17]: Should we autoupdate snapshots?

File: src/bin/cli.ts (hunk 1 above)
Comment (kahirokunn) [reply src/bin/cli.ts]: I Changed "file" option back to "optional"

File: package.json (hunk 3 above)
Comment (kahirokunn) [reply package.json]: sorry, I put it back.

**Comment by kahirokunn** [comment]
@msutkowski how about now?

**Review by msutkowski — APPROVED** [review]

File: src/bin/cli.ts (hunk 1 above)
Comment (msutkowski) [reply src/bin/cli.ts]: Thanks!

**Comment by msutkowski** [comment]
@kahirokunn Looks good to me. We'll see if @phryneas has any additional feedback. Awesome work! 💯

**Comment by kahirokunn** [comment]
Thx for your review😆

**Pull request merged**